.github/workflows/intermediate-docker.yml

```diff
@@ -24,6 +24,7 @@ on:
         options:
           - nightly-2023-12-03
           - nightly-2022-12-10
+          - nightly-2025-01-17
```

Comment:
Why this version specifically?

Reply:
I asked Greg about the reth's minimum version, he suggested 1.86, 1.86 corresponds nightly-2025-01-17

Reply:
Why use nightly and not 1.86 stable?